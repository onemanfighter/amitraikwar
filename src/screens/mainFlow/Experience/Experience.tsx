import { Box, Text } from '@chakra-ui/react';

const Experience = () => {
  return (
    <Box>
      <Text>Experience</Text>
    </Box>
  );
};

export default Experience;
