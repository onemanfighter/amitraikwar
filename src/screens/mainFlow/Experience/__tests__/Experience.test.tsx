import { render } from '@testing-library/react';
import Experience from '../Experience';

describe('Experience', () => {
  it('should render correctly', () => {
    const { container } = render(<Experience />);

    expect(container).toMatchSnapshot();
  });
});
