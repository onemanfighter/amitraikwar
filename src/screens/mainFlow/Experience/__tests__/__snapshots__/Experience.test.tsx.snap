// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience should render correctly 1`] = `
<div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-0"
    >
      Experience
    </p>
  </div>
</div>
`;
